android/buildAndroidBOINC-CI.sh

```diff
@@ -258,49 +258,44 @@ if [ $build_with_vcpkg = "yes" ]; then
     else
         triplets_setup="default"
     fi
-    packages="rappture curl[core,openssl]"
+    manifests="--x-manifest-root=vcpkg_json_$component/ --x-install-root=$VCPKG_ROOT/installed/"
     vcpkg_overlay="--overlay-ports=$vcpkg_ports_dir/ports --overlay-triplets=$vcpkg_ports_dir/triplets/$triplets_setup"
-    vcpkg_flags="$vcpkg_overlay --clean-after-build"
+    vcpkg_flags="$vcpkg_overlay  --feature-flags=versions --clean-after-build"
     if [ ! -d "$VCPKG_ROOT" ]; then
         mkdir -p $BUILD_DIR
         git -C $BUILD_DIR clone https://github.com/microsoft/vcpkg
     fi
     if [ ! -e /tmp/vcpkg_updated ]; then
         git -C $VCPKG_ROOT reset --hard
         git -C $VCPKG_ROOT pull
-        # ToDo: Remove when either https://github.com/openssl/openssl/issues/18059 or https://github.com/openssl/openssl/pull/18056 is fixed
-        git -C $VCPKG_ROOT checkout 89295c9fe22e97ca9e380a6c771ccf2ff09dca95
         $VCPKG_ROOT/bootstrap-vcpkg.sh
         touch /tmp/vcpkg_updated
     fi
     if [ $arch = "armv6" ]; then
         export ANDROID_NDK_HOME=$NDK_ARMV6_ROOT
 
-        $VCPKG_ROOT/vcpkg install $packages $vcpkg_flags --triplet=$(getTripletName $arch)
+        $VCPKG_ROOT/vcpkg install $manifests $vcpkg_flags --triplet=$(getTripletName $arch)
     fi
     if [ $arch = "arm" ]; then
         export ANDROID_NDK_HOME=$NDK_ROOT
 
-        $VCPKG_ROOT/vcpkg install $packages $vcpkg_flags --triplet=$(getTripletName $arch)
-        $VCPKG_ROOT/vcpkg install $packages $vcpkg_flags --triplet=$(getTripletName neon)
```

Comment:
Why neon is removed?

Reply:
Because each time it delete the previous triplet. In manifests mode currently cannot compile multiple triplets.
I can change the order, but I see this pointless.

Reply:
Then we need a new target (e.g. arm -neon) because I believe we still need this functionality. Otherwise it's a regression and need to be separately discussed.

Reply:
We never had it. it for different PR.

Reply:
Then why did you add it at all?